paper/paper.md

```diff
@@ -0,0 +1,432 @@
+three years, we have developed a curriculum and instructional format using
+Jupyter notebooks to effectively teach introductory Python for data science.
+This method, inspired by The Carpentries organization, uses bite-sized lessons
+followed by independent practice time to reinforce coding concepts, and
+culminates in a data science capstone project using real-world data.
+We believe our open curriculum is a valuable resource to the wider education
+community and hope that educators will use and improve our lessons, challenge
+problems, and teaching best-practices.
+Anyone can contribute to [our educational materials on GitHub](https://github.com/GWC-DCMB).
+
+
+# Statement of Need
+
+As women in the Department of Computational Medicine and Bioinformatics at the
+University of Michigan, we experience the gender gap in our field first hand.
+During the 1974-1975 academic year, women achieved 18.9% of total Bachelor's
+degrees in computer and information sciences in the US [@nces_digest_2012].
+By 1983-1984 this peaked at 37.1%, but fell to 17.6% by 2010-2011. We see this
+national trend in the training of the next generation of Bioinformaticians at
+Michigan Medicine.
+Since accepting its first students in 2001, the U-M Bioinformatics Graduate
+Program has graduated 66 male and 22 female doctorates as of 2019.
+This disparity begins at the applicant level — during 2016-2019 the average
+percentage of females applying directly to the Bioinformatics PhD program is
+35.2%, and 41% for female applicants listing Bioinformatics as first or second
+choice in the biomedical PhD umbrella program PIBS.
+
+Previous research on women's educational experiences in STEM have produced
+various explanations for persistent gender disparities
+[@benbow_gender_2016].
+One explanation is that women in first year computer science courses rate
+themselves as being less experienced, prepared, or able to master the course
+material than the men [@fisher_undergraduate_1997].
+The majority of our organization's founding graduate students (all women) began
+coding in our undergraduate careers or later.
+We wanted to provide a safe place for local young women to begin coding in high
+school, and develop confidence in themselves and their computational skills
+before college.
+A national organization, Girls Who Code, whose mission and values aligned with
+ours, was founded in 2012.
+Girls Who Code's mission is to close the gender gap in technology
+[@saujani_girls_2015].
+Because of our personal experiences and the paucity of women in our field, we
+began a student organization at the University of Michigan in 2017.
+For the past four academic years we have registered annually as a recognized
+Girls Who Code Club because the national organization provided name recognition,
+curriculum resources, guidance for a Capstone Impact Project, and a framework
+for launching a coding club.
+In 2019 we launched a Data Science Summer Experience which is unaffiliated with
```

Comment:
Could we list a reason on why we launch Summer Experience? What gap were we trying to fill?

Reply:
Help wanted! Anyone want to try to address this? The gist is something like: most of our club participants are from A2 & surrounding schools which have lots of resources, meanwhile we're nearby Detroit which has fewer resources.

Reply:
Done